File: dec11.c
//(11) The Robot Returns
#include<stdio.h>
#include<string.h>
int main(){
    char path[100];
    printf("enter path:");
    scanf("%s",&path);
    int count=0,i=strlen(path);
    while(i!=-1){
        if(path[i]=='u') count=count+1;
        else if(path[i]=='d') count=count-1;
        else if(path[i]=='r') count=count+2;
        else if(path[i]=='l') count=count-2;
        i=i-1;
    }
    if(count==0) printf("robot came to original position");
    else printf("robot didn't reach it's original position");
}

File: December 11/dec11.py
s = str(input("Enter string:"))
flag=0
flag1=0
if s.count("U")==s.count("D"):
    flag=1
if s.count("L")==s.count("R"):
    flag1=1
if flag==1 and flag1==1:
    print("True")
else:
    print("False")
